server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/JamesMailetContext.java

```diff
@@ -53,7 +53,6 @@
 import org.apache.james.server.core.MailImpl;
 import org.apache.james.user.api.UsersRepository;
 import org.apache.james.user.api.UsersRepositoryException;
-import org.apache.mailet.HostAddress;
```

Comment:
Why do you remove this import?

Reply:
because it is deprecated. I replace it by a FQDN so that I don't need to suppress warning deprecated on all the class but just on the needed methods.